packages/bits/src/lib/tabgroup/tab-heading/tab-heading.component.less

```diff
@@ -36,6 +36,16 @@
         }
     }
 
+    .nui-tab-heading:focus-within {
+        .focus-outline-inset();
+    }
+
+    .nui-tab-heading.active:focus-within {
+        .tab-link {
+            background: transparent;
+        }
+    }
+
```

Comment:
Could these be folded into the existing `.nui-tab-heading` block above? Like the following:

```less
.nui-tab-heading {
        white-space: nowrap;
        &.active {
            > .tab-link {
                .setCssVariable(color, nui-color-text-default);
                .setCssVariable(background-color, nui-color-bg-content);
            }
            border-left: none;
            border-bottom: @nui-line-thick solid;
            .setCssVariable( border-bottom-color, nui-color-selected-contrast);

            // here
            &:focus-within {
                .tab-link {
                    background: transparent;
                }
            }
        }
        .tab-link {
            .setCssVariable(background, nui-color-bg-transparent);
            .setCssVariable(color, nui-color-text-secondary);
            .setCssVariable(line-height, nui-line-height-default);
            .setCssVariable(font-weight, nui-font-weight-semibold);
            .setCssVariable(font-size, nui-font-size-default);
            display: block;
            padding: @nui-space-sm @nui-space-md;
            text-transform: uppercase;
            min-height: (@nui-space-sm * 4) - @nui-line-thick;
            cursor: pointer;
            &:hover {
                .setCssVariable(background, transparent);
            }
        }

        // and here
        &:focus-within {
            .focus-outline-inset();
        }
    }
```

It might be worth a try :)

Reply:
move focus-within above, thanks